features/netsocket/emac-drivers/TARGET_STM/mbed_lib.json

```diff
@@ -3,9 +3,41 @@
     "config": {
         "eth-rxbufnb": 4,
         "eth-txbufnb": 4,
-        "eth-phyaddr": {
+        "thread-stacksize": {
+            "help": "Stack size for stm32_emac_thread",
+            "value": 512
```

Comment:
how about increasing the value automatically when the trace is enabled? I think for an user it is difficult to guess a valid stacksize when it is configurable.

Reply:
During my tests, I didn't see any big difference...

Reply:
I think the adjustable stacksize come in from this:
https://github.com/ARMmbed/mbed-os/issues/12416#issuecomment-587036723
usually its a system thread and should have a fixed threadsize. Are there other cases than this special with trace enabled where stacksize can be a problem?